src/routes/categories/components/CategoryList.js

```diff
@@ -18,7 +18,9 @@ class CategoryList extends React.Component {
       children: PropTypes.arrayOf(PropTypes.instanceOf(CategoryModel)).isRequired
     })).isRequired,
     title: PropTypes.string,
-    content: PropTypes.string
+    content: PropTypes.string,
+    /** A search query to highlight in the categories titles **/
```

Comment:
I think you only need one `*` to close the comment tag for js-doc style 🤔 (`/** A search query to highlight in the categories titles */`)

Reply:
right.... thanks